spec/javascript/components/IndexTableCell.test.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import renderer from 'react-test-renderer'
+import IndexTableCell from 'components/IndexTableCell'
+
+// TODO: Add tests for editable cell functionality.
```

Comment:
I ended up limiting my tests here pretty severely. Let me know if you think it's worth it to add a few more